assets/configuration/core_configuration.json

```diff
@@ -15,22 +15,25 @@
       "enabled": true,
       "log_level": "INFO"
     },
-    "record_all_http_calls": true
+    "record_all_http_calls": false
```

Comment:
This should definitely be off. This should be off in production as well: https://github.com/ns8inc/protect-sdk-php/blob/master/assets/configuration/core_configuration.json